webviz_config/containers/_plot_a_table.py

```diff
@@ -0,0 +1,264 @@
+                    'options': self.columns,
+                    'value': self.plot_options.get('x', self.columns[0])
+                },
+                'y': {
+                    'options': self.columns,
+                    'value': self.plot_options.get('y', self.columns[0])
+                },
+                'z': {
+                    'options': self.columns,
+                    'value': self.plot_options.get('z', self.columns[0])
+                },
+                'size': {
+                    'options': self.numeric_columns,
+                    'value': self.plot_options.get('size', None)
+                },
+                'color': {
+                    'options': self.columns,
+                    'value': self.plot_options.get('color', None)
+                },
+                'facet_col': {
+                    'options': self.columns,
+                    'value': self.plot_options.get('facet_col', None)
+                },
+                'line_group': {
+                    'options': self.columns,
+                    'value': self.plot_options.get('line_group', None)
+                },
+                'barmode': {
+                    'options': ['stack', 'group', 'overlay', 'relative'],
+                    'value': self.plot_options.get('barmode', 'stack')
+                },
+                'barnorm': {
+                    'options': ['fraction', 'percent'],
+                    'value': self.plot_options.get('barnorm', None)
+                },
+                'histnorm': {
+                    'options': ['percent', 'propability', 'density',
+                                'propability density'],
+                    'value': self.plot_options.get('histnorm', None)
+                },
+                'trendline': {
+                    'options': self.numeric_columns,
+                    'value': None
+                },
+
+            })
+
+    def plot_option_layout(self):
+        '''Renders a dropdown widget for each plot option'''
+        divs = []
+        # The plot type dropdown is handled separate
+        divs.append(
+            html.Div(
+                style=self.style_options_div,
+                children=[
+                    html.P('Plot type'),
+                    dcc.Dropdown(
+                        id='{}-plottype'.format(self.plot_option_id),
+                        clearable=False,
+                        options=[{'label': i, 'value': i}
+                                 for i in self.plots.keys()],
+                        value=self.plot_options.get('type', 'scatter')
+                    )
+                ]
+            )
+        )
+        # Looping through all available plot options
+        # and renders a dropdown widget
+        for key, arg in self.plot_args.items():
+            divs.append(
+                html.Div(
+                    style=self.style_options_div,
+                    id='{}-div-{}'.format(self.plot_option_id, key),
```

Comment:
Use `f`-string as we are on Python3.6+:
`id=f'{self.plot_option_id}-div-{key}',`